### ISSUE TITLE
RSGNet

### ISSUE THREAD

**Issue by abc11223301** [Zizzzzzzz/SFNet_T-ITS24#9]
你的工作为我带来了很大的帮助，请问您可以分享一下rsgnet网络的代码吗？

**Comment by Zizzzzzzz** [comment]
We have released the code and pretrained weights for [RSGNet](https://github.com/Zizzzzzzz/SFNet_T-ITS24/tree/5a47b1679f7bf8bd36f95dc9b53fe6e87cc17580/RSGNet). However, due to inadvertent issues introduced during code maintenance, the current training pipeline may not fully reproduce the originally reported results, potentially due to differences in input normalization or loss function weighting.

**Comment by abc11223301** [comment]
> 我们已发布[RSGNet](https://github.com/Zizzzzzzz/SFNet_T-ITS24/tree/5a47b1679f7bf8bd36f95dc9b53fe6e87cc17580/RSGNet)的代码和预训练权重。然而，由于代码维护过程中引入的无意问题，当前的训练流程可能无法完全复现最初报告的结果，可能是由于输入归一化或损失函数权重的差异所致。

您好我已经申请访问了RSGNet权重文件的访问权限，请问可以同意申请吗，非常感谢。

**Comment by Zizzzzzzz** [comment]
Access is now open.
